.github/workflows/reusable-embed-deploy.yml

```diff
@@ -0,0 +1,66 @@
+name: Deploy Embed to Netlify
```

Comment:
The Netlify deployments differentiate between the projects, they do require generally running different steps and passing different env variables, so I wasn't able to have a reusable workflow for all Netlify deployments.